### ISSUE TITLE
[Feature Request] Zoom / Presentation mode

### ISSUE THREAD

**Issue by ruslanbes** [mRemoteNG/mRemoteNG#1066]
Sometimes I use the mRemoteNG to demonstrate things and the problem I'm facing is that the font is too small and nearly impossible to read on TV screens or projectors.

I tried the Windows Magnifying glass it's not very convenient and but the screen sharing app we use (coincidentally called "Zoom") doesn't support that.

It would be very useful to have a zoom feature like in IDEs and browsers: Zoom In/Out by Ctrl+ / Ctrl- (or Ctrl + Scroll). It's implemented simply by changing font size. Or something like a "presentation mode" in IntelliJ where the selected terminal fills the whole screen and massively increases the font size.

Or maybe is there a workaround I don't know?

**Comment by ruslanbes** [comment]
> Should this be zooming opened connections as well as the mRemoteNG UI?
Only opened connections. That's what I want to show.

> If you have an RDP connection opened, should it be zoomed?
Good question, I never used RDP, but I think it shouldn't be zoomed. If I demo something with RDP and it's too small to see I'd better use the native tools on the remote desktop to make them bigger.

**Comment by sparerd** [comment]
Ah ok, so in your case you would like this more for console-based connections like SSH?

**Comment by ruslanbes** [comment]
@sparerd yes, exactly. That's why I suggested it as simply increasing font size
